File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.cpp
class Solution {
public:
    int findMin(vector<int> &nums) {
        int l = 0, r = nums.size() - 1;

        int index;
        while( l <= r && l != index){
            index = (l+r) / 2;
            if(nums[r] < nums[index])
                l = index + 1;
            else
                r = index;
        }

        return nums[index];
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 1
        for index in range(1, len(nums)):
            if nums[index] != nums[k-1]:
                nums[k] = nums[index]
                k += 1
        
        return k

"""
1, 1, 2, 3, 4


"""